packages/itwinui-css/src/input/classes.scss

```diff
@@ -21,3 +25,34 @@
 .iui-end-icon {
   @include iui-end-icon;
 }
+
+.iui-input-flex-container {
+  @include iui-input-flex-container;
+  @include iui-input-size();
+
+  > input[type='search'] {
+    @include iui-search-box;
+  }
+
+  &[data-iui-size='small'] {
+    @include iui-input-size(small);
+  }
+
+  &[data-iui-size='large'] {
+    @include iui-input-size(large);
+  }
+
+  @each $status in positive, warning, negative {
+    &.iui-#{$status} {
```

Comment:
Using data attributes. Should InputFlexContainer have `aria-disabled`?

Reply:
no, because the container is a generic div.

`aria-disabled` is only applicable to specific roles listed [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled#associated_roles)